extensions/lifecycle/tasks/LifecycleDeleteObjectTask.js

```diff
@@ -47,7 +47,12 @@ class LifecycleDeleteObjectTask extends BackbeatTask {
         }, log, (err, blob) => {
             LifecycleMetrics.onS3Request(log, 'getMetadata', 'expiration', err);
             if (err) {
-                log.error('error getting metadata blob from S3', Object.assign({
+                // <!> Only in S3C <!> Backbeat API returns 'InvalidBucketState' error if the bucket is not versioned.
+                // In this case, instead of logging an error, it should be logged as a debug message,
+                // to avoid causing unnecessary concern to the customer.
+                // TODO: After the implementation of CLDSRV-461, we could remove this check.
```

Comment:
Let's have a ticket number with the TODO :slightly_smiling_face:

Reply:
Done here : https://github.com/scality/backbeat/pull/2541/files#diff-0a199af7ee10e4c59a4d84364fad46f84ce9c1cd9bffe7b55c54ed03eddb5166R53